backend/tilavarauspalvelu/models/payment_order/actions.py

```diff
@@ -109,3 +143,26 @@ def update_order_status(self, new_status: OrderStatus, payment_id: str = "") ->
 
         EmailService.send_reservation_confirmed_email(reservation=reservation)
         EmailService.send_reservation_confirmed_staff_notification_email(reservation=reservation)
+
+    def cancel_together_with_verkkokauppa(self) -> None:
```

Comment:
Does this need to be its own method? Could this logic just be included in `set_order_as_cancelled` 🤔

Reply:
This can likely be done, although there are cases where webshop should not be called, like in the webshop webhooks.

Reply:
Replaced some calls to `set_order_as_cancelled` with this and inlined some others with the contents of `set_order_as_cancelled`.